org.eclipse.wildwebdeveloper/src/org/eclipse/wildwebdeveloper/vue/VueLanguageServer.java

```diff
@@ -53,44 +60,56 @@ private void resolvePaths() throws IOException {
 		URL url = FileLocator
 				.toFileURL(getClass().getResource("/node_modules/@vue/language-server/bin/vue-language-server.js"));
 		vuePath = new File(url.getPath()).getAbsolutePath();
-		
-		url = FileLocator.toFileURL(getClass().getResource("/node_modules/typescript/lib"));
-		tsserverPath = new File(url.getPath()).getAbsolutePath();
+
 	}
+	
 
 	@Override
 	protected ProcessBuilder createProcessBuilder() {
 		ProcessBuilder builder = super.createProcessBuilder();
 		builder.environment().put("VUE_NONPOLLING_WATCHER", Boolean.toString(true));
+		builder.environment().put("NODE_ENV", "production");
 		return builder;
 	}
 
 	@Override
-	public Object getInitializationOptions(URI rootUri) {
-		Map<String, Object> options = new HashMap<>();
-		setWorkingDirectory(Paths.get(rootUri).toString());
-		
-		options.put("typescript", Collections.singletonMap("tsdk", tsserverPath));
-		options.put("diagnosticModel", 0);
-		options.put("additionalExtensions", new String[] {});
-		
-		Map<String, Object> legend = new HashMap<>();
-		legend.put("tokenTypes", new String[] {"component"} );
-		legend.put("tokenModifiers", new String[] {} );
-		options.put("semanticTokensLegend", legend);
-		
-		Map<String, Object> vue = new HashMap<>();
-		vue.put("hybridMode", false);
-		
-		options.put("vue", vue);
-		
-		return options;
+	public String toString() {
+		return "VUE Language Server: " + super.toString();
 	}
 
 	@Override
-	public String toString() {
-		return "VUE Language Server: " + super.toString();
+	public void handleMessage(Message message, LanguageServer languageServer, URI rootURI) {
```

Comment:
If someone have know howto move this to VueClientImpl I'll be happy to this this